src/api/Customer/getPaymentDetails.ts

```diff
@@ -0,0 +1,49 @@
+import { getData } from 'util/appConfigHelper';
+import fetchWithJWT from 'util/fetchHelper';
+import getApiURL from 'util/environmentHelper';
+import jwtDecode from 'jwt-decode';
+import { PaymentGateway } from 'redux/publisherConfigSlice';
+
+type PaymentDetail = {
+  active: boolean;
+  bound: boolean;
+  customerId: number;
+  id: number;
+  paymentGateway: PaymentGateway;
+  paymentMethod: string;
+  paymentMethodId: number;
+  paymentMethodSpecificParams: {
```

Comment:
not all payment methods will have such paymentMethodSpecificParams object. Eg. paypal will have only 'payerId' and 'holderName'